### PR TITLE
Fix localization

### PR DESCRIPTION
**Pull request by shu20031026**
## Summary
Fixed missing Japanese localization in the following locations.

### group creation banner

|before(en)|after(jp)|
|-|-|
|スクリーンショット 2024-01-12 午後5 05 18|スクリーンショット 2024-01-12 午後5 05 42|

### update buttons in drawer

|before(en)|after(jp)|
|-|-|
スクリーンショット 2024-01-12 午後5 07 43|スクリーンショット 2024-01-12 午後5 07 18

### popover button in table

|before(en)|after(jp)|
|-|-|
スクリーンショット 2024-01-12 午後5 08 34|スクリーンショット 2024-01-12 午後5 09 17

## notion
https://www.notion.so/cyberagent-group/000e93a674ef4f8f89187cb61b905311

## Checklist
- [x] [changeset file(s)](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) included

### CODE REVIEW

File: packages/app/src/pages/dashboard/groups/_/body/add/index.tsx [hunk 1]
```diff
@@ -63,28 +65,28 @@ const Add: React.FC<Props> = ({ onAdd, onCancel }) => {
         <div>
           <Head
             on={COLOR_SYSTEM.SURFACE}
-            title="Create a Group"
-            description="Manage endpoints in groups."
+            title={t('createGroup.head.title')}
+            description={t('createGroup.head.description')} //エンドポイントをグループで管理します。
```
Comment (nonoakij) [inline packages/app/src/pages/dashboard/groups/_/body/add/index.tsx:69]:
```suggestion
            description={t('createGroup.head.description')}
```

File: packages/app/src/pages/endpoints/[endpointId]/_/content/parts/action/index.tsx [hunk 2]
```diff
@@ -42,10 +43,22 @@ const Action: React.FC<Props> = ({ request, onClick }) => {
...
+        if (operation.summary) {
+          return operation.summary;
+        }
+        break;
+      case 'ja':
+        if (operation.description) {
+          return operation.description;
+        } else if (operation.summary) {
+          return operation.summary;
+        }
```
Comment (nonoakij) [inline packages/app/src/pages/endpoints/[endpointId]/_/content/parts/action/index.tsx:61]: Why do you need separate implementations for each language?

File: packages/app/src/pages/dashboard/groups/_/body/add/index.tsx (hunk 1 above)
Comment (shu20031026) [reply packages/app/src/pages/dashboard/groups/_/body/add/index.tsx]: @nonoakij  Sorry, I forgot to delete the comment out.

File: packages/app/src/pages/endpoints/[endpointId]/_/content/parts/action/index.tsx (hunk 2 above)
Comment (shu20031026) [reply packages/app/src/pages/endpoints/[endpointId]/_/content/parts/action/index.tsx:61]: @nonoakij 
In this part, it seems that the **"summary"** is obtained by referring to the data in `/example/golang/openapi/yaml/`

```yaml
  /users/{userId}:
    put:
      operationId: updateUser
      tags:
        - user
      summary: update user
      description: ユーザー更新
      parameters:
        - $ref: '#/components/parameters/UserIdPathParam'
      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/UserUpdatePayload'
      responses:
        204:
          description: OK
      security:
        - jwt: []
```

The Japanese item looked like **"description"**, so I branched out to get that data, and if **"description"** didn't exist, I implemented it to get **"summary"** as before.

Comment (nonoakij) [reply packages/app/src/pages/endpoints/[endpointId]/_/content/parts/action/index.tsx:61]: > The Japanese item looked like "description"

`description` in schema provided by you is in Japanese, but it does not need to be in Japanese.It is used to explain in more detail what the operation does.

Is it possible to remove splitting by language?

**Review by nonoakij — APPROVED** [review]

**Pull request merged**